aws/user_data/auto_scaled_ebs

```diff
@@ -0,0 +1,30 @@
+MIME-Version: 1.0
+Content-Type: multipart/mixed; boundary="==AUTO_SCALED_EBS_BOUNDARY=="
+
+--==AUTO_SCALED_EBS_BOUNDARY==
+Content-Type: text/cloud-config; charset="us-ascii"
+
+repo_update: true
+repo_upgrade: all
+
+packages:
+  - btrfs-progs
+  - bzip2
+  - git
+  - jq
+  - lvm2
+  - rsync
+  - sed
+  - unzip
+  - wget
+
+runcmd:
+  - curl -s "https://awscli.amazonaws.com/awscli-exe-linux-x86_64.zip" -o "/tmp/awscliv2.zip"
+  - unzip -q /tmp/awscliv2.zip -d /tmp && /tmp/aws/install
```

Comment:
Looking at the install location, it appears that the binary gets installed in `/usr/local/aws-cli` with symlinks to `/usr/local/bin`. Because of the way [nextflow mounts the aws directory]( https://www.nextflow.io/docs/latest/aws.html#aws-cli ), we might be able to get away with making a link from `/usr/local/aws-cli/aws` to `/usr/local/aws-cli/bin/aws`. This will result in nextflow mounting all of `/usr/local/aws-cli` which should not clobber anything. (If we were to specify the path to nextflow as `/usr/local/bin/aws` it would try to mount all of `/usr/local` in the docker images, which would probably be bad).

Reply:
Copied over to the https://github.com/AlexsLemonade/alsf-scpca/issues/169